src/App.js

```diff
@@ -1,11 +1,22 @@
 import "./App.css"
 import { Routes, Route } from "react-router-dom"
 import { Login } from "./components/auth/Login"
+import { Authorized } from "./views/Authorized"
+import { ApplicationViews } from "./views/ApplicationViews"
+import { Navbar } from "./navbar/Navbar"
 
 export const App = () => {
   return (
+    <>
+    <Navbar />
```

Comment:
Right now, having the Navbar component here works. Once we implement authentication, it may make sense to move the Navbar into ApplicationViews in the line just above <Outlet /> so that non-logged-in people will not see the navbar on the login and register screens. There is also the possibility that admins and non-admins will have different Navbar options that we may want to consider. Just a thought for future reference.